support/azure/azure-kubernetes/create-upgrade-delete/error-code-cnidownloadtimeoutvmextensionerror.md

```diff
@@ -10,7 +10,7 @@ ms.custom: sap:Create, Upgrade, Scale and Delete operations (cluster or nodepool
 ---
 # Troubleshoot the CniDownloadTimeoutVMExtensionError error code (41)
 
-This article discusses how to identify and resolve the `CniDownloadTimeoutVMExtensionError` error (also known as error code `ERR_CNI_DOWNLOAD_TIMEOUT`, error number 41) that occurs when you try to create and deploy a Microsoft Azure Kubernetes Service (AKS) cluster.
+This article discusses how to identify and resolve the `CniDownloadTimeoutVMExtensionError` error (also known as error code `ERR_CNI_DOWNLOAD_TIMEOUT`, error number `41` when uses linux distros or error number `35` to the windows nodes) that occurs when you try to create and deploy a Microsoft Azure Kubernetes Service (AKS) cluster.
```

Comment:
should we change to:
This article discusses how to identify and resolve the `CniDownloadTimeoutVMExtensionError` error (also known as error code `ERR_CNI_DOWNLOAD_TIMEOUT`, error number `41` when uses linux distros) or  error (also known as error code `WINDOWS_CSE_ERROR_DOWNLOAD_CNI_PACKAGE` , error number `35` to the windows nodes) that occurs when you try to create and deploy a Microsoft Azure Kubernetes Service (AKS) cluster. 


> $global:WINDOWS_CSE_ERROR_DOWNLOAD_CNI_PACKAGE=35 https://github.com/Azure/AgentBaker/blob/2a43850ca0495eadf9050e54d3292d3eeaf66049/parts/windows/windowscsehelper.ps1#L41

Reply:
@genlin what do you think

Reply:
I think making sense.It is necessary, if it happens to linux and windows nodepool.It should be visible to windows nodepool in this documentation.As per documented here (L41) -> the error description $global:WINDOWS_CSE_ERROR_DOWNLOAD_CNI_PACKAGE=35
 https://github.com/Azure/AgentBaker/blob/2a43850ca0495eadf9050e54d3292d3eeaf66049/parts/windows/windowscsehelper.ps1#L41